cloud_experts_tutorials/cloud-experts-entra-id-idp.adoc

```diff
@@ -113,16 +145,18 @@ $ APP_ID=yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyy <3>
 $ CLIENT_SECRET=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx <4>
 $ TENANT_ID=zzzzzzzz-zzzz-zzzz-zzzz-zzzzzzzzzzzz <5>
 ----
++
 --
 <1> Replace this with the name of your ROSA cluster.
-<2> Replace this with the name you used in the OAuth callback URL.
+<2> Replace this with the name you used in the OAuth callback URL that you generated earlier in this process.
 <3> Replace this with the Application (client) ID.
 <4> Replace this with the Client Secret.
 <5> Replace this with the Directory (tenant) ID.
 --
 
-Next, run the following command to configure the cluster's OAuth provider if you opted to enable group claims:
-
+. Configure the cluster's OAuth provider by running the following commands depending on if you opted to enable group claims or not.
+.. If you opted to enable group claims, run the following command:
```

Comment:
Nit: I think instead of a/b, these should be bullet points. 

https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#handling-nonsequential-actions

Another, smaller nit:

s/opted to enable/enabled
s/did not opt to enable/did not enable